client/src/styles/RankingPage.css

```diff
@@ -0,0 +1,107 @@
+.ranking-page-container {
+  margin: 15px 25px;
+}
+
+.ranking-lists-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.ranking-list-container {
+  width: 100%;
+}
+
+.ranked {
+  margin: 100px;
+}
+
+#ranking-page-graphic {
+  width: 40vw;
+  max-width: 200px;
+}
+
+.ranking-info-container {
+  display: flex;
+  flex-direction: row;
+  text-align: left;
+  justify-content: space-between;
+  margin-top: 20px;
+  margin-right: 20px;
+}
+
+.ranking-info {
+  font-size: 14px;
+}
+
+.info-icon-button {
+  padding: 0;
+   text-align: center;
+    height: 36px;
+    width: 36px;
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    background: none;
+    border: 2px solid black;
+    border-radius: 50%;
+    -moz-border-radius: 50%;
+    -webkit-border-radius: 50%;
+  margin-top: 4px;
+}
+
+.info-icon {
+  margin: auto;
+}
+
+.popup-container.ranking {
+  width: 200px;
+}
+
+.ranking-info-text {
+  margin: 0px;
+  font-size: 16px;
+}
+
+@media only screen and (min-width: 431px) and (max-width: 1024px) {
+  .ranking-page-container {
+    margin: 40px 10vw;
+  }
+  .popup-container.ranking {
+    width: 250px;
+  }
+  .ranking-info-text {
+    font-size: 18px;
+  }
+}
+
+@media only screen and (min-width: 431px) {
+  .info-icon-button {
+    margin-top: 10px;
+  }
+}
+
+@media only screen and (min-width: 550px) {
+  .ranking-info {
+    font-size: 16px;
+  }
+}
+
+@media only screen and (min-width: 1025px) {
+  .ranking-page-container {
+    box-sizing: border-box;
+    margin: 40px auto;
+    width: 60vw;
+  }
+  .ranking-lists-container {
+    flex-direction: row;
+  }
```

Comment:
The lists overflow on my desktop view, add: 
`.ranking-list-container { width: 50%; }`
or if you want a gap between the lists, then add something like this:
`.ranking-lists-container { flex-direction: row; gap: 5px; } .ranking-list-container { width: 49%; }`

Reply:
Done